react/features/base/participants/functions.js

```diff
@@ -45,3 +50,65 @@ function _getParticipants(participantsOrGetState) {
 
     return participants || [];
 }
+
+/**
+ * Returns the URL of the image for the avatar of a particular participant
+ * identified by their id and/or e-mail address.
+ *
+ * @param {string} participantId - Participant's id.
+ * @param {Object} [options] - The optional arguments.
+ * @param {string} [options.avatarUrl] - Participant's avatar url.
+ * @param {string} [options.email] - Participant's email.
+ * @param {string} [options.avatarId] - Participant's avatar id.
+ * @returns {string} The URL of the image for the avatar of the participant
+ * identified by the specified participantId and/or email.
+ *
+ * @public
+ */
+export function getAvatarURL(participantId, options = {}) {
+    if (typeof config === 'object' && config.disableThirdPartyRequests) {
```

Comment:
I've just moved this function from our old code to keep the logic the same. Unfortunately I'm not sure what's the logic behind this function and it's going to be hard for me to add inline comments.